src/VotingMultipliers.sol

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.22;
+
+import {IVotingMultipliers, IMultiplier} from "src/interfaces/IVotingMultipliers.sol";
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+/// @title VotingMultipliers
+/// @notice A contract for managing voting multipliers
+/// @dev Implements IVotingMultipliers interface
+contract VotingMultipliers is OwnableUpgradeable, IVotingMultipliers {
+    /// @notice Array of allowlisted multiplier contracts
+    IMultiplier[] public allowlistedMultipliers;
+
+    /// @notice Calculates the total multiplier for a given _user
```

Comment:
Drop the underscore form `_user` when referring to it in the comments to be consistent with other comments.

Reply:
addressed with a73cf97